src/preset.rs

```diff
@@ -234,3 +234,161 @@ impl<I: IntoBindings> IntoBindings for Bidirectional<I> {
         positive.chain(negative)
     }
 }
+
+/// A preset to map buttons as 3-dimensional input.
+///
+/// Uses [`SwizzleAxis`] and [`Negate`] to bind inputs to the Y and Z directions.
+///
+/// See also [`Axial`] and [`Bidirectional`].
```

Comment:
Let's point to more related presets.
```suggestion
/// See also [`Cardinal`] and [`Bidirectional`].
```